### PR TITLE
[author] Add OverlappingMarkerSpiderfier-Leaflet@0.2.6 w/ git auto-update

### LINKED ISSUE #10893: [Request] Add OverlappingMarkerSpiderfier-Leaflet (sister item of current cdnjs repo)
**Library name:**  OverlappingMarkerSpiderfier-Leaflet
**Git repository url:** https://github.com/jawj/OverlappingMarkerSpiderfier-Leaflet
**npm package name or url** (if there is one): n/a
**License (List them all if it's multiple):**  This software is released under the MIT licence.
**Official homepage:**  https://github.com/jawj/OverlappingMarkerSpiderfier-Leaflet
**Wanna say something? Leave message here:**  This repo is a clone of https://github.com/jawj/OverlappingMarkerSpiderfier which is already hosted by you. Both are by the same author, with the -leaflet version adding the functionality to a leafleat mapset rather than a google mapset. The author actually links this repo in the original repo. 

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by jawj**
Closes #10893 

# Profile of the lib
 * Git repository (required): https://github.com/jawj/OverlappingMarkerSpiderfier-Leaflet
 * License and its reference: MIT, http://www.opensource.org/licenses/mit-license.php
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 16
   - Count of stars: 152
   - Count of forks: 40

# Essential checklist
 * [X] I'm the author of this library
   * [X] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by jawj** [comment]
The built JS files are held in the `gh-pages` branch of the repo, but I'm not clear how I can specify that branch in the `autoupdate` section of `package.json`.

I'm also not clear how the autoupdate system is going to figure out version numbers ...

Is this really necessary? The files haven't changed in years and I have no changes planned right now.

**Comment by PeterDaveHello** [comment]
@jawj As we mention in the doc, we use git tag to recognize each version, have you read it yet? This is necessary yes, if need help, we can help you.

**Comment by jawj** [comment]
I did read the doc, but the tagging reference was opaque to me. What/how do I tag as required?

Also, how can I specify a non-master branch?

**Comment by PeterDaveHello** [comment]
@jawj I'm not sure what do you mean opaque, let me know the details please.

**Comment by PeterDaveHello** [comment]
@jawj Use "v0.0.1" as tag for version "0.0.1", "v1.2.3" for "1.2.3", etc.

It's totally fine to tag on non-master branch, it's not about branch.

@pvnr0082t please involve help @jawj in this, thanks.

**Comment by jawj** [comment]
Think this is ready now?

File: ajax/libs/OverlappingMarkerSpiderfier-Leaflet/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jawj/OverlappingMarkerSpiderfier-Leaflet.git",
+    "fileMap": [
+      {
+        "basePath": "bin",
+        "files": [
+          "oms.js",
+          "oms.min.js"
```
Comment (pvnr0082t) [inline ajax/libs/OverlappingMarkerSpiderfier-Leaflet/package.json]: @jawj It's more flexible to use `"oms*"` instead of these two.  Thanks.

Comment (jawj) [inline ajax/libs/OverlappingMarkerSpiderfier-Leaflet/package.json]: It's even more flexible to use "*". These are all the files. Shall I do that?

**Comment by PeterDaveHello** [comment]
This PR needs to be cleaned up ...

**Comment by PeterDaveHello** [comment]
@pvnr0082t can you help @jawj squash and rebase those commits?

**Comment by PeterDaveHello** [comment]
@pvnr0082t you need reviews ...

**Comment by maruilian11** [comment]
@PeterDaveHello The PR has been approved by the reviewers.

**Pull request merged**